### DISCUSSION TITLE
Very basic general knowledge

### DISCUSSION THREAD

**Discussion by Dark-Base** [community/community#68994] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

How to secure any account from phishing/hacking/cyber bullying or any other ?

**Comment by RitamPaul** [comment]
use 2 factor authentication

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
